Terminal.Gui/Views/HexView.cs

```diff
@@ -312,9 +312,13 @@ public override bool ProcessKey (KeyEvent keyEvent)
 				MoveUp (bytesPerLine);
 				break;
 			case Key.Tab:
-				leftSide = !leftSide;
-				RedisplayLine (position);
-				firstNibble = true;
+				if (keyEvent.IsCtrl) {
```

Comment:
If you included this change to `HexView.cs` in this PR because you thought it fixed the "can't tab out of hex view" bug (#531), I don't think it does (I just pulled this PR down and tried it and the `Unicode` scenario still shows #531 reproduces.

Reply:
With ctrl + tab moves to the next View. Now in the scenario where only HexView is located, the cursor goes to a place that I no longer had the head to investigate.

Reply:
@tig I think we should not break the functionality of the Tab key that `HexView `uses internally which is to move between hexadecimal code and text. As in a text editor, you press the `Tab `key to insert it into the text and to move to another `View `you will have to press `Ctrl + Tab` or `Shift + Tab` to move to the next or previous `View`, respectively.

Reply:
On Windows, at least, if an Edit control is in a dialog, along with other controls, `Tab` tabs to next control, and `Shift-Tab` tabs to previous. You have to use `Ctrl-Tab` to enter a tab. If the Edit control is the only control, `Tab` adds a tab to the text and `Sift-Tab` does nothing. 

I think we should follow the same behavior for all views that accept input in Terminal.Gui.

Reply:
I think it is debatable because if I am in a text editor I would like to use all the available keys to work on that text and not move to other controls, but anyway ...
Of course, whoever created this control also thought of the same.

Reply:
> I think it is debatable because if I am in a text editor I would like to use all the available keys to work on that text and not move to other controls, but anyway ...
> Of course, whoever created this control also thought of the same.

I authored a contributors guide that has some tenets for making decisions like this. It's *ALL* debatable, but instead of debating specific examples like this in isolation, I'd prefer we debate the tenets first (using examples like this to pressure test the tenets). https://github.com/migueldeicaza/gui.cs/blob/master/CONTRIBUTING.md

Let me know what you think. Please use a new Issue if you have comments on the CONTRIBUTING.md.